cloudwash/providers/aws.py

```diff
@@ -120,19 +120,22 @@ def dry_ocps():
                             resources=instances,
                         ):
                             dry_data["OCPS"]["delete"].extend(cluster_resources)
+                            dry_data["OCPS"]["clusters"].extend([cluster_name])
                     else:
                         # For resources with no associated EC2 Instances, identify as leftovers
-                        dry_data["OCPS"]["delete"].extend(
-                            filter_resources_by_time_modified(
-                                time_threshold, resources=cluster_resources
-                            )
+                        filtered_leftovers = filter_resources_by_time_modified(
+                            time_threshold, resources=cluster_resources
                         )
+                        if filtered_leftovers:
+                            dry_data["OCPS"]["delete"].extend(filtered_leftovers)
+                            # Print cluster_name only if it's resources are deletable
+                            dry_data["OCPS"]["clusters"].extend([cluster_name])
```

Comment:
Ahh! not something we are looking for adding into the list of clusters.

Still if you want to print the clusters, You could add that to the line saying: 

```
Resources from the region: us-east-1
```

as 

```
Resources from the region: us-east-1 and clusters: 'ods-qe-aws-01-pool-zn-qdns7', 'testgsno-pool-shph9-pcbfk'
```

Reply:
Reverted, will be resolved in the resource deletion PR (WIP)